File: 0101_0200/0168/0168.py
with open("input.txt", "r") as f, open("output.txt", "w") as q:
    n = f.readline().strip()
    s = ""
    for i in range(1, 10001):
        s += str(i)
    q.write(str(s.find(n) + 1))

File: 0101_0200/0168/0168.go
package main

import (
	"fmt"
	"os"
	"strconv"
	"strings"
)

func main() {
	f, _ := os.Open("input.txt")
	q, _ := os.Create("output.txt")
	var n int
	fmt.Fscan(f, &n)
	var sb strings.Builder
	sb.WriteString("123456789")
	for i := 10; i < 10001; i++ {
		sb.WriteString(strconv.Itoa(i))
	}
	fmt.Fprint(q, strings.Index(sb.String(), strconv.Itoa(n))+1)
}
